gulpfile.babel.js

```diff
@@ -42,6 +42,9 @@ const meta = require( './package.json' );
 import {} from './tools/builder/frontend-css';
 import {} from './tools/builder/admin-css';
 
+// These paths should alawys be ignored when watching files
+const alwaysIgnoredPaths = [ '!node_modules', '!node_modules/**', '!vendor', '!vendor/**', '!docker/**' ];
```

Comment:
@Viper007Bond do you know why we had `'!folder', '!folder/**'` instead of just `'!folder/**'`?

https://github.com/Automattic/jetpack/pull/8656/files#diff-ba210a9157252cc983268fdc3aa3ed52R336

Reply:
Nope, I just copied the format that was used for `node_modules`.

Reply:
I'm inclined to do the same... :-D thanks

Reply:
Tested that these should be safe to remove. Gulp relies to [`node-glob`](https://github.com/isaacs/node-glob).

Running this from jetpack folder should return `docker` so that folder (and anything inside it) is already excluded with just `docker/**`:
```bash
npm i -g --quiet glob && \
node -e "require( 'glob' )('docker/**', {}, function (e, paths) { console.log( paths[0] ) })" && \
npm un -g --quiet glob
```